uploader.json

```diff
@@ -1811,6 +1811,16 @@
             "pattern": [],
             "symbolic_name": "logrotate_conf"
         },
+        {
+            "file": "/var/lib/config-data/gnocchi/etc/gnocchi/gnocchi.conf",
+            "pattern": [],
+            "symbolic_name": "gnocchi_conf"
+        },
+        {
+            "file": "/etc/gnocchi/gnocchi.conf",
+            "pattern": [],
+            "symbolic_name": "gnocchi_conf"
```

Comment:
@chenlizhong you will need to add the filter from your plugin to the specs in this PR.